tmva/tmva/CMakeLists.txt

```diff
@@ -431,11 +433,8 @@ ROOT_STANDARD_LIBRARY_PACKAGE(TMVA
     ${installoptions}
 )
 
-if(builtin_vdt)
-  # cannot export include for vdt, as it points to build dir
+if(vdt OR builtin_vdt)
   target_include_directories(TMVA PRIVATE ${VDT_INCLUDE_DIRS})
```

Comment:
If users build against TMVA (say they `#include` a TMVA header`) - should they be able to find the VDT headers? Yes they should: `tmva/tmva/src/DNN/Architectures/Cpu/ActivationFunctions.hxx` includes them. But that means that these headers *should* be `PUBLIC` and `ActivationFunctions.hxx` cannot be used by users without VDT. @lmoneta what's the plan here?

Reply:
It is only for ROOT compilation itself.
After ROOT is build, vdt includes should be found in `$ROOTSYS/include` or in some system directories. Probably, very special case when `vdt` installed in custom paths, must be covered in the user makefiles.
But anyway - I do not see how ROOT cmake files can help users to build their code - they are not available at all when user code is build.

Reply:
The ActivationFunctions.hxx is a header included only in the source file, it is not public. Also the vet header is protected by the pre-processor macro R__HAS_VDT. If it is enables the vdt header should be visible, correct ?

Reply:
Lorenzo is right. These includes are only used in source directories and not provided to `$ROOTSYS/include` directory. And even then - vdt will be available in `$ROOTSYS/include/vdt` subdirectory - and can be used after checking `R__HAS_VDT`

@Axel-Naumann Once again - this PR is only about compilation of ROOT itself. 
There should be no changes for users